libraries/chain/webassembly/privileged.cpp

```diff
@@ -182,8 +182,9 @@ namespace eosio { namespace chain { namespace webassembly {
       for (auto& f: finalizers) {
```

Comment:
I think we should use  `std::unordered_set<eosio::bls_g1, g1_hash, g1_equal> unique_finalizer_keys;` above like in the bios contract.

Reply:
Will change.

Reply:
Done by https://github.com/AntelopeIO/leap/pull/1989/commits/b9e464b744606a9658ab3be27370726dc2850d88

Reply:
We decided it would be easier to just stick with `std::set`.